### PR TITLE
Disable apistub gen artifact gen for python pipeline

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Python APIStub generator has a new change that seems to have broken apistubgen for azure-core-tracing-opencensus. This PR is to temporarily remove this apistub gen step since we haven't rolled out change to utilize this artifact yet.

### CODE REVIEW

**Comment by weshaggard** [comment]
@praveenkuttappan instead of disabling is it easy enough to rollback the version of the apistub tool we use? Also is there any way to try to test/check for these breaks when new changes are made to apistub?

**Comment by praveenkuttappan** [comment]
We will add a pipeline to test this tool against all python packages.

**Review by scbedd — APPROVED** [review]

**Review by sima-zhu — APPROVED** [review]
The code is not super complicated. Better to remove or roll back.

**Comment by praveenkuttappan** [comment]
> @praveenkuttappan instead of disabling is it easy enough to rollback the version of the apistub tool we use? Also is there any way to try to test/check for these breaks when new changes are made to apistub?

@tjprescott has submitted a PR to fix actual issue in apistubgen. This PR is to unblock any pipeline failures until that's merged.

**Comment by praveenkuttappan** [comment]
This is a temporary change to unblock python team.

**Pull request merged**
